docs/configuration_file.md

```diff
@@ -171,6 +173,12 @@ Some options are also mandatory.
 
 * `build_from` (*optional*, `string`) — build source to use, consists of 2 parts separated with delimiter ':', first part can be : image or imagestream, and second part is corresponding image or imagestream
 
+### `[platform:ARCH]` options
+
+* `architecture` (*optional*, `string`) — platform's GOARCH (Go language platform name). If not declared, this option assumes the value of `ARCH`.
+
+* `enable_v1` (*optional*, `boolean`) — enable support for Docker Registy HTTP API v1 for a given platform. Only one platform should have this option enabled. Defaults to false.
```

Comment:
Typo: Registy -> Registry

Reply:
Thanks, I keep forgetting to run aspell on documentation changes. Got another typo on the run and included the fix in this patch.